docs/what-is-bitcoin.md

```diff
@@ -0,0 +1,71 @@
+company, or an individual. This made it easy for these trusted persons to block people's money,
+or succumb to Government intervention. Satoshi figured out a way to overcome that, by inventing
+a data structure called the blockchain, which is distributed between all users in a peer-to-peer network.
+
+Users of the Bitcoin network create transactions, and then digitally sign them with cryptographic
+signatures. A transaction contains all the information necessary to send the bitcoins to its recipient.
+Once the transaction is ready, it gets broadcasted to the Bitcoin peer-to-peer network so that it can
+be included into the blockchain by the Bitcoin miners, a group of users and businesses who use special
+hardware to solve really complex puzzles to secure transactions and extend the blockchain. But what
+really is the blockchain?
+
+## Blockchain
+
+The blockchain is a chain of blocks, created by the Bitcoin miners by solving computationally
+difficult puzzles using their specialised hardware. Each block is generated approximately every
+ten minutes, and contains transactions between users. Miners compete to solve blocks so they can
+claim the block reward and transaction fees. The block reward is a payout given to the miner that
+solves the block, the payout amount halves every four years and is the way Bitcoin bootstraps its
+circulating supply.
+
+Miners listen for incoming transactions between users, place them into blocks, and then compete
+with one another to solve the computationally challenging puzzle. The first miner to solve the
+puzzle announces their block to the network, and that block gets added to the distributed blockchain.
+Once the block is added, other miners then abandon the old block they were working on, and create a
+new block which builds on top of the recently announced block. This process builds a chain of blocks,
+the blockchain. Every two weeks, the network uses an algorithm to limit the rate of block creation
+to an average of one block every ten minutes.
+
+## Fees and the role of miners
+
+The amount of space available for transactions in each block is limited, so users pay transaction
+fees to bid for space in the next block for their transaction. This means Bitcoin's transaction fees
+can be higher during periods of heavy use, and lower during quieter times. The transaction fees are
+always changing, but most Bitcoin wallet software uses smart algorithms to figure out the correct fee
+to pay to get your transaction processed by the miners. It costs nothing to receive Bitcoin, the
+transaction fee is only paid by the sender.
+
+The transaction fee goes to the miner who mined the block the transaction was included in. It's
+important to understand that even though miners collect a fee, and create blocks, they do not
+control the Bitcoin network. The blocks miners create are verified by the users who run the
+Bitcoin software. Amongst other things, the software checks any new blocks to make sure the miner
```

Comment:
"the Bitcoin software" might leave the user wondering what "the software is".  Maybe say "Bitcoin verification software".  Currently the top two search results for that query via Google (without quotes) are both on Bitcoin.org.